### ISSUE TITLE
Use system-consistent icon sizing.

### ISSUE THREAD

**Issue by RokeJulianLockhart** [digitaltrails/vdu_controls#65]
The current iconography is unnecessarily large:

> ![Screenshot_20231017_190229](https://github.com/digitaltrails/vdu_controls/assets/42837531/4cfec485-3cff-4256-bb05-8f7672c53b82)

See https://discuss.kde.org/t/qml-python3-how-to-read-icons-kcm-configured-sizes/1209/7?u=rokejulianlockhart

**Comment by digitaltrails** [comment]
- Could this be related to #63?  
- Does it involve a High DPI desktop or a normal one?

**Comment by digitaltrails** [comment]
In #63 I added a setting to try and help adjust icons appropriately according to DPI.  However, I've never be sure if what I've done is the right approach.  

It now occurs to me that the choice of icon sizes might be more accurately determined by interrogating the default font size in use.  

I've had to play similar games in the past. To detect a dark/light theme I create some text in a QLabel and test the relationship between the text and background colors.  In this instance could extract a QLabel's font size (in a relevant metric) and base icon sizes around that.

**Comment by RokeJulianLockhart** [comment]
> Does it involve a High DPI desktop or a normal one?

https://www.infobyip.com/detectmonitordpi.php states 96×96 DPI. I use a 1440p monitor.

**Comment by digitaltrails** [comment]
In the `settings dialog`, try turning off the *adjust for dpi* setting (it will prompt to restart the app). 

I think the following code might be returning True:
```
def is_high_dpi() -> bool:
    return adjust_for_dpi and QApplication.primaryScreen().physicalDotsPerInch() >= 160
```
I'll have to add some logging to see for sure.

**Comment by RokeJulianLockhart** [comment]
> In the settings dialog, try turning off the adjust for dpi setting

I've not been able to locate it. Could you screenshot the window you refer to?

**Comment by digitaltrails** [comment]
![image](https://github.com/digitaltrails/vdu_controls/assets/5510901/c8ee1570-c2c5-48b7-bce1-ab11f51b0168)

**Comment by RokeJulianLockhart** [comment]
@digitaltrails, mine is different.

![Screenshot_20231017_224145](https://github.com/digitaltrails/vdu_controls/assets/42837531/7a23704d-20c8-4123-aeda-24abdc284302)

**Comment by digitaltrails** [comment]
Oh, it's a new option in v.12.0, you'd need to probably download the vdu_controls.py from github and replace the version you're using - or wait for the distribution to catch up.

I thought I'd released in in v1.11, but no v1.12 only got released yesterday.

**Comment by digitaltrails** [comment]
Duplicate of #63 - see further info there.
